packages/ui-tailwind/src/components/Block.tsx

```diff
@@ -0,0 +1,11 @@
+export type BlockProps = {
+  children?: React.ReactNode;
+  margin?: number;
```

Comment:
Instead of numbers, can you get the tailwind margins? I'm assuming m-12.745 doesn't exist but 12.745 is a valid number, does tailwind have types published?

Reply:
Yep, its gonna be tailwind types just like text size & colours👌